client/app/screens/InviteList.js

```diff
@@ -83,7 +84,16 @@ export default class InviteListScreen extends React.Component {
           });
       });
 
-    // Initial load of invites
+    let reloadInvites = function(invites) {
+      this.setState({ dataSource: ds.cloneWithRows(invites) });
```

Comment:
It looks like you're reusing the code on line 78 — I'd extract it to a class method `_loadInvites` or similar and call it above and in `onDefaultMessage`.